sources/cfsignal/src/cloudformation.rs

```diff
@@ -29,15 +30,33 @@ pub async fn signal_resource(
         .load()
         .await;
 
-    // TODO: add support for HTTP Proxy
     #[cfg(feature = "fips")]
     let crypto_mode = CryptoMode::AwsLcFips;
     #[cfg(not(feature = "fips"))]
     let crypto_mode = CryptoMode::AwsLc;
 
-    let http_client = HyperClientBuilder::new()
-        .crypto_mode(crypto_mode)
-        .build_https();
+    let https_proxy: Option<String> = match env::var_os("HTTPS_PROXY") {
```

Comment:
style nit: `match` statements like this are more idiomatically written using [`Option::map`](https://doc.rust-lang.org/std/option/enum.Option.html#method.map).

Reply:
```suggestion
    let https_proxy = env::var("HTTPS_PROXY").ok();

    let no_proxy = env::var("NO_PROXY")
        .ok()
        .map(|n| n.split(',').map(|s| s.to_string()).collect::<Vec<_>>());
```
So in practice like this. It does look clean, but since we already have approvals I think we can save it for next time we're in the crate or if `clippy` flags it.